wandb/sdk/wandb_require.py

```diff
@@ -46,22 +41,11 @@ def require_service(self) -> None:
         self._require_service()
 
     def require_core(self) -> None:
-        if is_require_legacy_service():
-            raise UnsupportedError(
-                'Cannot use wandb.require("core") because legacy-service'
-                " is already required--did you use"
-                ' wandb.require("legacy-service") or set the'
-                " WANDB__REQUIRE_LEGACY_SERVICE environment variable?"
-            )
-        os.environ[_REQUIRE_CORE] = "true"
+        wandb.termwarn(
+            "This is currently the default behavior and does not need to be set."
```

Comment:
The user won't know where this message is coming from, so it should be more explicit.

> `wandb.require("core")` is redundant as it is now the default behavior.

Reply:
yep i was just looking at the wording here and i didn't like it, i'll update to your wording, thanks